sphinx-doc/figures/md-walls.ipynb

```diff
@@ -168,7 +168,7 @@
     "energy = lj_wall_energy(r, 1.1, 3.0)\n",
     "energy_base = lj_wall_energy(r_without, 0.0, 3.0)\n",
     "\n",
-    "fig = plt.Figure(figsize=(10, 6.18), dpi=100)\n",
+    "fig = plt.Figure(figsize=(7, 4.32624056), dpi=100)\n",
```

Comment:
We could consider adding a `matplotlibrc` file that sets all these defaults so we can just `fig = plt.Figure` and `fig.savefig(fn)`

Reply:
We could. At this point, it would be extra work as it is likely we will not rerun these notebooks for several years.